docs/pages/database-access/guides/ha.mdx

```diff
@@ -4,14 +4,58 @@ description: How to configure Teleport database access in a Highly Available (HA
 ---
 
 You can deploy the Teleport Database Service in a Highly Available (HA)
-configuration in a couple of common ways: combined replicas and separate
-replicas. Both of those revolve around pointing multiple Database Services to
-the same database instance.
+configuration using two methods, both of which involve pointing multiple
+Database Service instances to the same database instance:
 
-## Combined replicas
+- Deploying multiple Database Service instances with the same dynamic resource
+  watchers 
+- Deploying multiple Database Service instances configured to proxy the same
+  database instances
```

Comment:
Should we punctuate these bullet points?

Reply:
I'm okay either way as long as punctuation is consistent. I've added punctuation.